### PR TITLE
Event: Add basic unit tests for event aliases

### PR DESCRIPTION
**Pull request by dmethvin**
I don't think we need anything fancy here. A few questions inline.

### CODE REVIEW

File: test/unit/event.js [hunk 1]
```diff
@@ -2816,6 +2816,28 @@ QUnit.test( "originalEvent property for Chrome, Safari, Fx & Edge of simulated e
...
+
+	jQuery.each( aliases, function( i, name ) {
+
+		if ( !jQuery.fn[ name ] ) {
+			assert.ok( true, "no method present for jQuery.fn." + name );
+			return;
+		}
+
+		// e.g. $(elem).click(...).click();
+		jQuery( document.body )[ name ]( function( event ) {
```
Comment (gibson042) [inline test/unit/event.js]: An element under `#qunit-fixture` would probably be better than `document.body`, to make leaks even less likely.

**Comment by dmethvin** [comment]
All feedback incorporated. Unfortunately it's not possible at the moment to run all the event tests because they make scattered use of event aliases themselves. I opened #2691 for that.

**Comment by gibson042** [comment]
LGTM.

**Comment by mzgol** [comment]
LGTM

**Pull request closed without merging**